docs/hosting/deploy.md

```diff
@@ -1,20 +1,417 @@
+
+### Environment Variables
+
+When deploying to Reflex's hosting service, the command prompt asks if you want to add any environment variables. These are encrypted and safely stored. We recommend that backend API keys or secrets are entered as `envs`. Make sure to enter the `envs` without any quotation marks.
+
+The environment variables are key value pairs. We do not show the values of them in any CLI commands, only their names (or keys). However, if your app intentionally prints the values of these variables, the logs returned still contain the printed values. At the moment, the logs are not censored for anything resembling secrets. Only the app owner and Reflex team admins can access these logs.
+
+You access the values of `envs` by referencing `os.environ` with their names as keys in your app's backend. For example, if you set an env `ASYNC_DB_URL`, you are able to access it by `os.environ["ASYNC_DB_URL"]`. Some Python libraries automatically look for certain environment variables. For example, `OPENAI_API_KEY` for the `openai` python client. The `boto3` client credentials can be configured by setting `AWS_ACCESS_KEY_ID`,`AWS_SECRET_ACCESS_KEY`. This information is typically available in the documentation of the Python packages you use.
+
+### Updating Deployment
+
+To redeploy or update your app, navigate to the project directory and type `reflex deploy` again. This command communicates with the hosting service to automatically detects your existing app by the same name. This time the deploy command overwrites the app. You should see a prompt similar to `Overwrite deployment [ app-name ] ...`. This operation is a complete overwrite and not an incremental update.
+
+## Hosting CLI Commands
+
+All the `reflex` commands come with a help manual. The help manual lists additional command options that may be useful. You type `--help` to see the help manual. Some commands are organized under a `subcommands` series. Here is an example below. Note that the help manual may look different depending on the version of `reflex` or the `reflex-hosting-cli`.
+
+```python eval
+doccmdoutput(
+    command="reflex deployments --help",
+    output="""Usage: reflex deployments [OPTIONS] COMMAND [ARGS]...
+
+  Subcommands for managing the Deployments.
+
+Options:
+  --help  Show this message and exit.
+
+Commands:
+  build-logs  Get the build logs for a deployment.
+  delete      Delete a hosted instance.
+  list        List all the hosted deployments of the authenticated user.
+  logs        Get the logs for a deployment.
+  regions     List all the regions of the hosting service.
+  status      Check the status of a deployment.
+"""
+)
+```
 
-## Reflex Deploy
+`--loglevel` is another common command option. When setting `--loglevel debug`, a command prints additional information, which can be helpful during debug.
```

Comment:
might be nice to show this command in action with an example (like the one above)

Reply:
did you mean "--loglevel debug"? There's probably not much to see and varies a lot based on what the command is. On a second thought, going to remove this instead. This flag helps us debug issues rather than for end user